cfn-resources/auditing/test/inputs_1_invalid.json

```diff
@@ -1,6 +1 @@
-{
-  "ProjectId": "",
-  "Profile": "default",
-  "AuditAuthorizationSuccess": "false",
-  "AuditFilter": 123124235
-}
+
```

Comment:
It appears the files are not fully deleted, has happened to me before

Reply:
wow, thanks, i don't how this happened. thank for noticing, I didn't event notice in this GH file compare page

Reply:
there is a bug in the git hook we should fix it 😭

Reply:
thanks, I'll take care of it